CONTRIBUTING.md

```diff
@@ -22,11 +22,11 @@ Before you submit a PR, make sure that:
 - You code doesn't raise any ESlint warning
 - Your changes are explained in your PR
 
-When in doubt, [Create a new issue](https://github.com/LooksRare/sdk-v2/issues/new) to discuss your proposal first.
+When in doubt, [Create a new issue](https://github.com/hypercerts-org/marketplace-sdk/issues/new) to discuss your proposal first.
 
 ### Release
 
 - Create a [personal access token](https://github.com/settings/tokens/new?scopes=repo&description=release-it) (Don't change the default scope)
```

Comment:
Are we using release-it?

Reply:
Removed the entire file as it's not up to date with our contribution guidelines.